mmdet/datasets/utils.py

```diff
@@ -1,10 +1,13 @@
 from collections import Sequence
-
+import copy
```

Comment:
It may look better with the following import order.

```python
import copy
from collections import Sequence
[blank line]
import xxx
```

Reply:
OK.